packages/playwright-test/src/runner/dispatcher.ts

```diff
@@ -267,6 +267,8 @@ export class Dispatcher {
       }
       const { result, steps } = runData;
       const parentStep = params.parentStepId ? steps.get(params.parentStepId) : undefined;
+      // Sanitize location that comes from user land, it might have extra properties.
```

Comment:
I can see where you toot it from, but I am not sure it is necessary, should be Ok to pass it in as is.

Reply:
Do you mean I just don't have to remove the `function` property or do you also mean I don't need to filter out a location whose `file` includes `'@playwright'`?